### ISSUE TITLE
UTF-8/diacritics

### ISSUE THREAD

**Issue by Pumpapa** [hemanth/node-rsj#5]
Hi,

Is this the right place to log a question for node-rsj?
I'm having trouble with diacritics. 
When I write the output of 
```
rsj.r2j(url,function(json) {
                        fs.appendFileSync('xxxx',json);
```
I get something containing `Handel met Iran stuit vooral op barri�res`
whereas the source has: `Handel met Iran stuit vooral op barrières`.

The default encoding for appendfilesync is utf8.

Any suggestions on how to proceed?

**Comment by hemanth** [comment]
Yes, this is the right place, the problem is with `JSON.strigify` we need to look into ways to maintain the unicode values.
